tests/compile-fail/weak_memory/racing_mixed_size_read.rs

```diff
@@ -16,6 +15,9 @@ fn split_u32_ptr(dword: *const u32) -> *const [u16; 2] {
     unsafe { std::mem::transmute::<*const u32, *const [u16; 2]>(dword) }
 }
 
+// Racing mixed size reads may cause to two loads to read-from
```

Comment:
```suggestion
// Racing mixed size reads may cause two loads to read-from
```

Reply:
One day I'll write a grammarly extension for clippy 🤦